### DISCUSSION TITLE
Mobile Clients Should Have Neutral Gender Option

### DISCUSSION THREAD

**Discussion by Simon818** [BearWare/TeamTalk5#2504] [Ideas]
### Description

On iOS, there are only male and female selector buttons. on Android, the gender setting is a checkbox. It should ideally be possible to change the gender to neutral and remove the icon altogether. This is possible on the desktop client already, but not on the phone.

### Application

- [ ] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by amirmahdifard** [comment]
actually i was thinking to suggest to remove the neutral gender completely because anyway, and people can select this randomly in teamtalk, and those that are ok to select their right gender. for this reason I think that gender checkbox in android makes sence and all platforms should become like this because checked for femail, not checked for mail. what everyone thinks about this?

**Comment by Simon818** [comment]
I think people use the gender option as a form of identification, and some people don't want to identify as either male or female, and since the protocol allows this, we should support it in the client. I personally know people who want a neutral gender option on mobile.
